### ISSUE TITLE
Unicode input not compatible with some software

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#4]
```
When you have Unicode characters in a Phrase, the replacement doesn't 
always work as expected. For example, I have a Phrase replacing alpha,, 
with α.

*In gedit, this works perfectly.
*In Opera's address bar, it just inserts 03b1, the Unicode codepoint for 
the lowercase alpha symbol.
*In an Opera edit box, it removes the abbreviation and then opens the 
source viewer. This is caused presumably by it sending Ctrl+U to attempt 
to convert the "03b1" to the Unicode character. Ctrl+U is used by Opera to 
toggle source code viewing.

I'm using version 60.6, on Ubuntu Jaunty.


```

Original issue reported on code.google.com by `john.j.b...@gmail.com` on 1 Oct 2009 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
Another problem with Unicode support: I have another Phrase for inserting the 
Greek 
character nu (ν). I have nu in the Phrase window, but when the Phrase is run 
and the 
text inserted, I get a Latin lowercase "gee" (g). This is not a problem with 
the 
font, as I can still paste nu into the document I'm working in with Ctrl-V, and 
it 
comes out correctly.

The attached file shows the result of using the Phrase in gedit.
```

Original comment by `john.j.b...@gmail.com` on 4 Oct 2009 at 4:41

Attachments:
- [Screenshot-2.png](https://storage.googleapis.com/google-code-attachments/autokey/issue-4/comment-1/Screenshot-2.png)

**Comment by GoogleCodeExporter** [comment]
```
I noticed this bug is produced exclusively in QT4 applications, and in most 
cases, a
carriage return seems to be sent after the unicode codepoint of the intended 
symbol.
In GTK apps it never seems to fail. I haven't tested other widget sets.
```

Original comment by `Hetdegon` on 11 Dec 2009 at 7:35

**Comment by GoogleCodeExporter** [comment]
```
My work around:
text = u"žluťoučké \n štěňátko"
clipboard.fill_clipboard(text)
keyboard.send_keys("<ctrl>+v")

```

Original comment by `daniel.b...@gmail.com` on 18 Dec 2009 at 4:53

**Comment by GoogleCodeExporter** [comment]
```
The entry method autokey uses for unicode characters will only work in 
applications
based on the GTK toolkit - hence why you note it does not work in QT4 apps. The
solution is as you have seen, to send the text via the clipboard. There is no 
other
way around this.
```

Original comment by `cdekter` on 24 Mar 2010 at 11:20
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in v0.80.0
```

Original comment by `cdekter` on 19 Sep 2011 at 12:35
- Changed state: **Fixed**
